analytic_operating_unit/models/account_analytic_account.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
[Guideline](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L3) says agpl.
Not sure yet. Let me know if still it should be LGPL.

Reply:
>Guideline says agpl.

@sudhir-erpharbor It's Only for ref., But check in a base or other migrated modules.

I think It's LGPL, So change Other Files

Reply:
Please check the other [OCA 12](https://github.com/OCA/web/blob/12.0/web_disable_export_group/models/ir_http.py) modules as well.

Reply:
> Please check the other [OCA 12](https://github.com/OCA/web/blob/12.0/web_disable_export_group/models/ir_http.py) modules as well.

Yes @sudhir-erpharbor, I am not sure for licensing term.

@pedrobaeza @aheficent can you please help here.

Reply:
I did not see this comment before. The license was LGPL in previous versions and I think we should keep the same in this version.

Reply:
@aheficent [Sample module](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L3) says AGPL, so I changed it.
Let me know if you think it should be LGPL, I will change it.

Reply:
Yes, normally it is AGPL, but OCA allows authors to choose LPGL if they consider/prefer that. In this case all the operating unit repo modules are under LGPL so better to keep using the same license.

Reply:
Done

Reply:
Thanks! 

BTW, I got to do the git push --forced again because the same issue appeared.

For futuere changes please fetch the code from` sudhir-erpharbor:analytic_ou_12` and then do `git reset --hard sudhir-erpharbor:analytic_ou_12`